src/lapack/backends/rocsolver/rocsolver_scope_handle.hpp

```diff
@@ -0,0 +1,76 @@
+/***************************************************************************
+*  Copyright 2020-2022 Intel Corporation
+*  Copyright (C) Codeplay Software Limited
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  For your convenience, a copy of the License has been included in this
+*  repository.
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+**************************************************************************/
+#ifndef _ROCSOLVER_SCOPED_HANDLE_HPP_
+#define _ROCSOLVER_SCOPED_HANDLE_HPP_
+#if __has_include(<sycl/sycl.hpp>)
+#include <sycl/sycl.hpp>
+#else
+#include <CL/sycl.hpp>
+#endif
+#if __has_include(<sycl/backend/cuda.hpp>)
+#include <sycl/backend/cuda.hpp>
```

Comment:
Yeah, I agree that this header is not necessary. It was originally intended to be used in the "__has_include" check and somehow ended up in the inclusion as well. 
This will be removed in the next commit.